plugins/react/frontend/components/common/IFrame/index.js

```diff
@@ -4,8 +4,18 @@
 import React from 'react';
 import path from 'path';
 
-const createHtml = (componentPath, dest, userFiles, injectTags, commonsChunkFilename) => (
-  `<!DOCTYPE html>
+const createHtml = (
+  componentPath,
+  publicPath = '',
+  dest = '',
+  userFiles,
+  injectTags,
+  commonsChunkFilename
+) => {
+  const basePath = publicPath || dest ? path.join('/', publicPath, dest) : '';
```

Comment:
My idea was to move this to the webpack plugin and pass that down to the React plugin?